packages/@aws-cdk/aws-rds/lib/serverless-cluster.ts

```diff
@@ -106,11 +106,15 @@ export interface ServerlessClusterProps {
 
   /**
    * The VPC that this Aurora Serverless cluster has been created in.
+   *
+   * @default - No VPC related construct will be created:
+   * - If subnetGroup is not provided, no DB subnet group will be associated with the DB cluster.
+   * - If securityGroups is not provided, no VPC security groups will be associated with the DB cluster.
```

Comment:
Let's leave this description to `subnetGroup` and `securityGroups` properties, I don't think this is the correct place for them.

Reply:
👍🏻